app/experimenter/experiments/models/nimbus.py

```diff
@@ -73,6 +92,8 @@ class NimbusExperiment(NimbusConstants, models.Model):
         "NimbusBranch", blank=True, null=True, on_delete=models.CASCADE
     )
 
+    objects = ExperimentQueueManager()
```

Comment:
Is `ExperimentQueueManager` an appropriate name for this? Or should it just be `ExperimentManager`? Does it supersede the existing manager? Did some quick reading on it and I think that is true, but wanna be sure.

Reply:
So.... You can have multiple managers on a model, so like we could have 

```py
objects = ExperimentManager()
queued_objects = ExperimentQueueManager()
```

but we don't need multiple, so just one is fine, and if it's only one it's usually just called `<Model>Manager` so yeah `NimbusExperimentManager` is fine here.